generic_tracers/generic_COBALT.F90

```diff
@@ -5095,7 +5126,9 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
       endif !}
     enddo; enddo ; enddo  !} i,j,k
 
-    ! O2 saturation
+    ! Calculate the oxygen saturation using the relationships of Garcia and Gordon, 1992. Oxygen solubility in
```

Comment:
Garcia and Gordon equation 8 has a A3 * T_s^2 term that is missing below. Is this their mistake or ours?

Reply:
![Screenshot_20240605_100246](https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/ocean_BGC/assets/5852283/00646bd2-9f55-4b1d-980a-d8946ccc4ae4)

Reply:
Check value also matches if I use the COBALT code with the new coefficients from the paper.

```fortran
real, parameter :: a_0 = 2.00856
real, parameter :: a_1 = 3.224
real, parameter :: a_2 = 3.99063
real, parameter :: a_3 = 4.80299
real, parameter :: a_4 = 9.78188e-01
real, parameter :: a_5 = 1.71069
real, parameter :: b_0 = -6.24097e-03
real, parameter :: b_1 = -6.93498e-03
real, parameter :: b_2 = -6.90358e-03
real, parameter :: b_3 = -4.29155e-03
real, parameter :: c_0 = -3.1168e-07
```

Reply:
That was indeed their mistake, not ours.  Thanks for the careful checks!